introduction.md

```diff
@@ -1,2 +1,7 @@
 Hi, My favorite path is git of course, 'cause that's the only one I've exploredi but I'm gonna mention JavaScript for excercise sake.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?